### ISSUE TITLE
Mobile autoscroll is jumpy / jerky

### ISSUE THREAD

**Issue by ekerik220** [clauderic/dnd-kit#1366]
Hey, first of all thank you for all your work on this package, it's awesome.

I'm having an issue with the autoscroll on mobile where the scroll is jumpy / jerky, especially if I'm scrolling slowly by holding the dragging item near the boundary where the autoscroll kicks in. Here's a video showcasing this:

https://github.com/clauderic/dnd-kit/assets/5779535/cd25d99c-c7a8-46d1-8afd-e3e9ab84e507

However, using the same mobile phone, the storybook examples work flawlessly. So I think there must either be something wrong with the way I'm using the package, or some incompatible package versions or something. 🤔

[Here's a small repository](https://github.com/ekerik220/dnd-kit-auto-scroll-issue) featuring the example used in the above video. I followed an example from the [docs](https://docs.dndkit.com/presets/sortable#drag-overlay) to make it.

Can you think of anything that could be causing this?

**Comment by ekerik220** [comment]
Update on this. I noticed that if I use the "Open Canvas in new tab" button on the storybook examples the same issue is occurring... so I guess something about the storybook container is making it work properly?

**Comment by ekerik220** [comment]
Another update. I think there is some sort of bug with .scrollBy(). I've only been to test it on my own phone so I'm not sure how wide spread the bug is (I have an iPhone 13 Pro Max). It could very likely have something to do with the ProMotion screen.

I've found that adding this css seems to fix the issue, but it's not ideal, so would love to hear other solutions.

```
html {
  overflow-y: hidden;
}
body {
  overflow-y: auto;
  height: 100dvh;
}
```

I've created an [issue](https://stackoverflow.com/questions/78204899/frequent-small-window-scrollby-calls-causing-jittery-scroll-on-mobile) on stackoverflow as well to try and get some more insight.

It looks like this isn't an issue with this package specifically, although I suppose since this package is using scrollBy it is of concern to the package. Either way, I'll close this issue for now.
